app/src/main/java/org/schabi/newpipe/fragments/list/playlist/PlaylistFragment.java

```diff
@@ -471,12 +470,13 @@ private void updateBookmarkButtons() {
         }
 
         final int iconAttr = playlistEntity == null
-                ? R.attr.ic_playlist_add : R.attr.ic_playlist_check;
+                ? R.drawable.ic_playlist_add_white_24dp
+                : R.drawable.ic_playlist_add_check_white_24dp;
 
         final int titleRes = playlistEntity == null
                 ? R.string.bookmark_playlist : R.string.unbookmark_playlist;
 
-        playlistBookmarkButton.setIcon(ThemeHelper.resolveResourceIdFromAttr(activity, iconAttr));
+        playlistBookmarkButton.setIcon(iconAttr);
         playlistBookmarkButton.setTitle(titleRes);
```

Comment:
While you are at it, you could move the variables above directly here:
```suggestion
        playlistBookmarkButton.setIcon(playlistEntity == null
                ? R.drawable.ic_playlist_add_white_24dp
                : R.drawable.ic_playlist_add_check_white_24dp);
        playlistBookmarkButton.setTitle(playlistEntity == null
                ? R.string.bookmark_playlist
                : R.string.unbookmark_playlist);
```